subprojects/plugins/src/integTest/groovy/org/gradle/api/tasks/JavaExecDebugIntegrationTest.groovy

```diff
@@ -26,6 +27,7 @@ import org.junit.Rule
 import spock.lang.Ignore
 import spock.lang.Issue
 
+@LocalOnly(because = "we observed timeouts when running on remote executors")
```

Comment:
I think we have a much better explanation like this. We can also link to the issue.